tests/test_spatialpops.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import pytest
+
+from laser_core.demographics.spatialpops import distribute_population_skewed
+from laser_core.demographics.spatialpops import distribute_population_tapered
+
+
+def test_distribute_population_skewed_basic():
+    np.random.seed(42)
+    result = distribute_population_skewed(1000, 5, 0.3)
+    assert sum(result) == 1000
+    assert len(result) == 5
+    assert abs(sum(result[:2]) / 1000 - 0.8) <= 0.1
+
+
+def test_distribute_population_skewed_alternative():
+    np.random.seed(42)
+    result = distribute_population_skewed(500, 3, 0.4)
+    assert sum(result) == 500
+    assert len(result) == 3
+    assert abs(sum(result[:1]) / 500 - 0.6) <= 0.1
+
+
+def test_distribute_population_skewed_zero_nodes():
+    with pytest.raises(ValueError, match="Number of nodes must be greater than 0"):
+        distribute_population_skewed(1000, 0, 0.3)
```

Comment:
It might be useful here and below to use named parameters, e.g.,

```python
distribute_population_skewed(tot_pop=1000, num_nodes=0, frac_rural=0.3)
```